nss-day-cohort-45/tabloidfullstack-gibbering-gizmos
# client side works now

How to test:

Run Tabloid server side, then run React side and select "Categories" in the navbar.
Make sure the category cards show up with edit and delete buttons (which are obviously not functional yet)
